CHANGELOG.md

```diff
@@ -59,6 +59,7 @@ in the naming of release branches.
 
 ### Changed
 
+- Changed the representaion of  key deposits. Deleted the field dsDeposits. Moved it to the UMap. Added some new functions to UMapCompact module to accomodate changes.
```

Comment:
I'm almost always a huge fan of adding to the changelog! I think in this case, though, the details are so internal that no one besides the ledger team would notice (outside of the reduced memory consumption and the structure of the ledger state when dumped from the node). I'd say leave this line out, but happy to hear if others disagree!